### PR TITLE
Update TomcatWebXml for 1.x

### PR DESCRIPTION
**Pull request by xiangce**
- move it to `tomcat_xml.py`, since there are several `xml` can be
  handle in this file
- update documentation

### CODE REVIEW

**Comment by xiangce** [comment]
This is a temporary PR, I will update it after the base class made by @zhangyi733 is merged.

**Comment by xiangce** [comment]
Since #855 is merged and I updated this PR as per it, @zhangyi733 @bfahr please take a look at this one when you are free, thanks.

File: insights/parsers/tomcat_xml.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    Examples:
+        >>> type(web_xml)
+        <class 'insights.parsers.tomcat_xml.TomcatWebXml'>
+        >>> web_xml.get('session-timeout') == 30
+        True
+    """
+
+    TIMEOUT_KEYWORD = ''
+
+    def _parse_dom(self):
```
Comment (bfahr) [inline insights/parsers/tomcat_xml.py]: Before we start using this method, I think we should consider whether it should be a public method in the base class.  Currently its private so its not included in any of the documentation and a user would not know that its there and could be implemented in a child class.  Now that I understand how its used I think the base class should be changed to make this a public method, and the documentation should be clear about how to use it.

Comment (xiangce) [reply insights/parsers/tomcat_xml.py]: Thanks @bfahr , that makes sense to us. We'll make it public.

**Pull request merged**